grails-datastore-core/src/main/groovy/org/grails/datastore/mapping/config/ConfigurationBuilder.groovy

```diff
@@ -188,7 +188,11 @@ abstract class ConfigurationBuilder<B, C> {
 
                         def buildMethod = ReflectionUtils.findMethod(newBuilder.getClass(), 'build')
                         if (buildMethod != null) {
-                            method.invoke(builder, buildMethod.invoke(newBuilder))
+                            try {
+                                method.invoke(builder, buildMethod.invoke(newBuilder))
+                            } catch(Throwable e) {
+                                log.error("build method threw exception")
```

Comment:
could we add `log.error("build method threw exception", e)` so the user can see the stack trace & message?